README.md

```diff
@@ -189,10 +189,8 @@ $ make install
 To build with [CMake][]:
 
 ```bash
-$ mkdir -p build
-
-$ (cd build && cmake .. -DBUILD_TESTING=ON) # generate project with tests
-$ cmake --build build                       # add `-j <n>` with cmake >= 3.12
+$ cmake -S . -B build -DBUILD_TESTING=ON # generate project with tests
```

Comment:
I think you do t need the -S . if you are in the source tree already.

Reply:
Done!